src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -18,26 +18,61 @@ namespace Microsoft.PowerShell
     /// <summary>
```

Comment:
@daxian-dbw, your last commit had 2 failures in `PowerShell-CI-windows`
Resources strings in Microsoft.PowerShell.Commands.Utility (was -ResGen used with Start-PSBuild).'UnblockFileStrings' should be an available type and the strings should be correct
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ResourceValidation\TestRunner.ps1: line 58
58:                 $resourceType | Should -Not -BeNullOrEmpty
```
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Exception calling "Invoke" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 136
136:                 $ps.AddScript('$pid').Invoke() | Should -Be $pwshId

```